cas/ctg_store.c

```diff
@@ -1665,8 +1671,11 @@ static int ctg_exec(struct m0_ctg_op    *ctg_op,
 {
 	int ret = M0_FSO_AGAIN;
 
-	ctg_op->co_ctg = ctg;
-	ctg_op->co_ct = CT_BTREE;
+	/* Do not overwrite ctg_op params if co_ct == CT_DEAD_INDEX. */
```

Comment:
it will be good if we have new func for dead ctg ; this looks to be specific to catalogue btree

Reply:
Added a separate function.